packages/vuetify/src/mixins/intersectable/index.ts

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue'
+import { consoleWarn } from '../../util/console'
+
+export default function intersectable (options: { onVisible: string[] }) {
+  if (typeof window === 'undefined' || !('IntersectionObserver' in window)) {
+    return Vue.extend({
+      name: 'intersectable',
+      // do nothing because intersection observer is not available
+    })
+  }
+
+  return Vue.extend({
+    name: 'intersectable',
+
+    mounted () {
+      const observer = new IntersectionObserver((
+        entries: IntersectionObserverEntry[] = [],
```

Comment:
Don't need this, the type is inferred and correct me if I'm mistaken but isn't `entries` always defined?